Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[workspace]
+members = [
+    "mako_rollup"
+]
```

Comment:
This is necessary for the Rust Makefile's `check` target to find `mako_rollup`.

Reply:
You should make sure someone with more Rust clue signs off on this, then.